src/main/java/com/google/api/generator/gapic/composer/Composer.java

```diff
@@ -106,4 +108,21 @@ private static GapicClass generateGenericClass(Kind kind, String name, Service s
             .build();
     return GapicClass.create(kind, classDef);
   }
+
+  @VisibleForTesting
+  protected static List<GapicClass> addApacheLicenseToGapicClassList(
```

Comment:
For consistency with other similar methods, just `addApacheLicense` so we can change the argument type without changing the method name.

Reply:
Done.